src/python/pants/engine/fs_test.py

```diff
@@ -1003,3 +1003,27 @@ def is_changed_snapshot() -> bool:
         )
 
     assert try_with_backoff(is_changed_snapshot)
+
+
+# -----------------------------------------------------------------------------------------------
+# Comparison and representation of Digests
+# -----------------------------------------------------------------------------------------------
+
+
+def test_digest_repr() -> None:
+    assert (
+        str(Digest("ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff", 1))
```

Comment:
This is hard to read. Maybe use f * 50 or whatever the relevant thing is.

Reply:
Done